cmd/kubeapps-apis/plugins/kapp_controller/packages/v1alpha1/server.go

```diff
@@ -73,13 +74,10 @@ type Server struct {
 // the k8s client config.
 func NewServer(configGetter server.KubernetesConfigGetter) *Server {
 	return &Server{
-		clientGetter: func(ctx context.Context) (dynamic.Interface, error) {
+		clientGetter: func(ctx context.Context, cluster string) (dynamic.Interface, error) {
 			if configGetter == nil {
 				return nil, status.Errorf(codes.Internal, "configGetter arg required")
 			}
-			// The Kapp Controller plugin currently supports interactions with
```

Comment:
why is it that now we support other clusters?

Reply:
I'm not seeing any reason why we wouldn't... we use the users cred to whatever cluster is in the context. As long as kapp-controller is installed there, it should work fine (but haven't tested it). I needed to update this because the UI always sends the cluster with the request anyway. If there's a problem we can make it conditional (I'll check it at some point).